src/scss/skin-modern/components/_subtitleoverlay-cea608.scss

```diff
@@ -0,0 +1,29 @@
+// sass-lint:disable force-element-nesting
```

Comment:
Please avoid disabling lint rules that can easily be fulfilled. That should only be done in cases where there is no (easy) solution.